src/pod/personal-storage.ts

```diff
@@ -48,47 +45,72 @@ export class PersonalStorage {
   async create(name: string): Promise<Pod> {
     assertActiveAccount(this.accountData)
     name = name.trim()
-    let lookupAnswer: LookupAnswer | undefined
-    let list: Pod[]
-    try {
-      lookupAnswer = await getFeedData(
-        this.accountData.connection.bee,
-        POD_TOPIC,
-        prepareEthAddress(this.accountData.wallet!.address),
-        this.accountData.connection.options?.downloadOptions,
-      )
-      list = extractPods(lookupAnswer.data.chunkContent())
-    } catch (e) {
-      list = []
-    }
+    const podsInfo = await getPodsList(
+      this.accountData.connection.bee,
+      prepareEthAddress(this.accountData.wallet!.address),
+      this.accountData.connection.options?.downloadOptions,
+    )
 
-    assertPodsLength(list)
-    assertPodNameAvailable(list, name)
+    const nextIndex = podsInfo.pods.length + 1
+    assertPodsLength(nextIndex)
+    assertPodNameAvailable(podsInfo.pods, name)
 
     let epoch: Epoch
     const currentTime = getUnixTimestamp()
 
-    if (lookupAnswer) {
-      epoch = lookupAnswer.epoch.getNextEpoch(currentTime)
+    if (podsInfo.lookupAnswer) {
+      epoch = podsInfo.lookupAnswer.epoch.getNextEpoch(currentTime)
     } else {
       epoch = getFirstEpoch(currentTime)
     }
 
-    const index = list.length + 1
-    const newPod = { name, index } as Pod
-    list.push(newPod)
-    const allPodsData = stringToBytes(list.map(item => `${item.name},${item.index}`).join('\n') + '\n')
+    const newPod = { name, index: nextIndex } as Pod
+    podsInfo.pods.push(newPod)
+    const allPodsData = podListToBytes(podsInfo.pods)
     const wallet = this.accountData.wallet!
     // create pod
     await writeFeedData(this.accountData.connection, POD_TOPIC, allPodsData, wallet.privateKey, epoch)
     // create root directory for pod
     const now = getUnixTimestamp()
     const path = '/'
     // create a new key pair from the master mnemonic. This key pair is used as the base key pair for a newly created descendant pod
-    const pathWallet = Wallet.fromMnemonic(wallet.mnemonic.phrase, `m/44'/60'/0'/0/${index}`)
+    const pathWallet = Wallet.fromMnemonic(wallet.mnemonic.phrase, `m/44'/60'/0'/0/${nextIndex}`)
```

Comment:
what is the `m/44'/60'/0'/0/` prefix?

Reply:
BIP44 derivation path. The last part is the wallet index, which can be extracted from the mnemonic phrase. I think this is the reason why pods start counting from 1 and not from 0. The wallet with index 0 stores the encrypted mnemonic phrase, and all derivatives of this wallet are responsible for storing pods.